Sofa/Component/LinearSolver/Iterative/src/sofa/component/linearsolver/iterative/PCGLinearSolver.cpp

```diff
@@ -19,21 +19,21 @@
 *                                                                             *
 * Contact information: contact@sofa-framework.org                             *
 ******************************************************************************/
-#define SOFA_COMPONENT_LINEARSOLVER_ITERATIVE_SHEWCHUKPCGLINEARSOLVER_CPP
-#include <sofa/component/linearsolver/iterative/ShewchukPCGLinearSolver.inl>
+#define SOFA_COMPONENT_LINEARSOLVER_ITERATIVE_PCGLINEARSOLVER_CPP
+#include <sofa/component/linearsolver/iterative/PCGLinearSolver.inl>
 #include <sofa/component/linearsolver/iterative/MatrixLinearSolver.inl>
 #include <sofa/core/ObjectFactory.h>
 #include <sofa/component/linearsystem/MatrixFreeSystem.h>
 
 namespace sofa::component::linearsolver::iterative
 {
 
-void registerShewchukPCGLinearSolver(sofa::core::ObjectFactory* factory)
+void registerPCGLinearSolver(sofa::core::ObjectFactory* factory)
 {
     factory->registerObjects(core::ObjectRegistrationData("Linear system solver using the Shewchuk conjugate gradient iterative algorithm.")
-        .add< ShewchukPCGLinearSolver<GraphScatteredMatrix, GraphScatteredVector> >());
+        .add< PCGLinearSolver<GraphScatteredMatrix, GraphScatteredVector> >());
```

Comment:
According to the [guidelines](https://sofa-framework.github.io/doc/contributing-to-sofa/lifecycle-mechanisms/#renaming-a-component) an alias must be created.